### PR TITLE
Added UG document for AIAssistView Control template

### PR DESCRIPTION
**Pull request by RavikumarAkuthotaSf4250**
Added UG document for AIAssistView Control template

### CODE REVIEW

File: MAUI/AIAssistView/control-template.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+{% tabs %} 
+{% highlight c# %} 
+
+   public class CustomAIAssiststView : SfAIAssistView
+   {
+       public CustomAIAssiststView() { }
+
+       protected override AssistViewChat CreateAssistChat()
+       {
+           return new CustomassistViewchat(this);
```
Comment (Lakshminatarajan) [inline MAUI/AIAssistView/control-template.md:96]: Add comments here as return the custom AssistViewChat.

File: MAUI/AIAssistView/control-template.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+        Content = assistView;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## CreateAssistChat Override in AI AssistView
+
+The `CreateAssistChat` method allows for the customization of the chat view functionality within the AI AssistView control. By overriding this method, can create their own custom implementation of the chat view, allowing for greater control over the appearance and behavior of chat interactions.It provides the flexibility to modify how chat messages are displayed, how user interactions are handled. Here’s how to override the `CreateAssistChat` method to return a custom instance of `AssistViewChat`.
```
Comment (Lakshminatarajan) [inline MAUI/AIAssistView/control-template.md:85]: The `CreateAssistChat` method allows for the customization of the chat view functionality within the AI AssistView control. By overriding this method, can create their own custom implementation of the chat view, allowing for greater control over the appearance and behavior of chat interactions. It provides the flexibility to modify how chat messages are displayed, how user interactions are handled. Here’s how to override the `CreateAssistChat` method to return a custom instance of `AssistViewChat`.

Comment (RavikumarAkuthotaSf4250) [reply MAUI/AIAssistView/control-template.md:85]: updated as per request

File: MAUI/AIAssistView/control-template.md (hunk 1 above)
Comment (RavikumarAkuthotaSf4250) [reply MAUI/AIAssistView/control-template.md:96]: added the comment

File: MAUI/AIAssistView/control-template.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+The `CustomAssistViewChat `class inherits from `AssistViewChat` and can be used to further customize the chat view.
+
+{% tabs %} 
+{% highlight c# %} 
+
+   public class CustomassistViewchat : AssistViewChat
+   {
+      public CustomassistViewchat(SfAIAssistView assistView) : base(assistView)
+       {
+           this.ShowMessageInputView = false;
```
[the remark this answers is not in the document]
Comment (RavikumarAkuthotaSf4250) [reply MAUI/AIAssistView/control-template.md:112]: Added the comments

**Pull request closed without merging**